cola/models/stash.py

```diff
@@ -104,9 +107,36 @@ def __init__(self, context, stash_name, keep_index):
 
     def do(self):
         if self.keep_index:
-            args = ['save', '--keep-index', self.stash_name]
+            args = ['push', '--keep-index', '-m', self.stash_name]
         else:
-            args = ['save', self.stash_name]
+            args = ['push', '-m', self.stash_name]
```

Comment:
`git stash save` wasn't deprecated until Git 2.16.0, and versions before 2.13 did not have that command at all.

Please add a version check (see `git grep version.check`) with a new `stash-push` entry for '2.16.0' in `cola/version.py` so that we can support old and new git versions.  That way we can use `git stash push` for 2.16+ only.

Reply:
I see… I'm going to look into the matter, the philosophy to keep supporting older versions is pretty relevant!